README.md

```diff
@@ -35,75 +35,104 @@ ctags --fields=afmikKlnsStz readtags.c  readtags.h
 **Opening Tags File**
 ```python
 import ctags
-from ctags import CTags, TagEntry
+from ctags import CTags
 import sys
 
 try:
     tagFile = CTags('tags')
-except:
+except OSError as err:
+    print err
```

Comment:
Yep

On 17 May 2016 at 17:42, Matthieu Gautier <notifications@github.com> wrote:

> In README.md
> <https://github.com/jonashaag/python-ctags3/pull/6#discussion_r63547497>:
>
> >  import sys
> >
> >  try:
> >      tagFile = CTags('tags')
> > -except:
> > +except OSError as err:
> > +    print err
>
> The "except OSError as err" is python3 syntax.
> Do you speak about the print function ?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/jonashaag/python-ctags3/pull/6/files/540b4bfd9dda55f4b823652dc507d5cbe9c60621#r63547497>
>